src/c++/perf_analyzer/genai-perf/genai_perf/llm_metrics.py

```diff
@@ -439,6 +439,10 @@ def __init__(
         tokenizer: AutoTokenizer,
     ) -> None:
         self._tokenizer = tokenizer
+        # disable add_bos_token so that llama-tokenizer does not add bos token
```

Comment:
Could you write out what bos stands for in the comment?

Reply:
edited comment